torch/testing/_internal/common_quantization.py

```diff
@@ -2930,6 +2930,21 @@ def forward(self, x1, x2, x3, x4):
             w = torch.cat([z, y])
             return w
 
+    class Conv2dWithSplit(torch.nn.Module):
+        def __init__(self) -> None:
+            super().__init__()
+            self.conv1 = torch.nn.Conv2d(3, 3, 3)
+            self.conv2 = torch.nn.Conv2d(3, 3, 3)
+
+        def forward(self, x):
+            x = self.conv1(x)
+            x1, x2 = x.split(2, dim=1)
```

Comment:
Can you add a comment explaining why `split` is important? Is it because it returns a list/tuple of tensors?

Reply:
yes